recipes/r-bwstools/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set version = '1.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bwstools
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/bwsTools_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/bwsTools/bwsTools_{{ version }}.tar.gz
+  sha256: 1cdff4461a031b28238db5471db5387bca53cb27dfd819eb687e7a906faa37a0
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-crossdes
+    - r-dplyr
+    - r-igraph
+    - r-magrittr
+    - r-rlang
+    - r-tidyr
+  run:
+    - r-base
+    - r-crossdes
+    - r-dplyr
+    - r-igraph
+    - r-magrittr
+    - r-rlang
+    - r-tidyr
+
+test:
+  commands:
+    - $R -e "library('bwsTools')"           # [not win]
+    - "\"%R%\" -e \"library('bwsTools')\""  # [win]
+
+about:
+  home: https://github.com/markhwhiteii/bwsTools
```

Let's also include the OSF site they reference from GitHub:

```suggestion
  home: https://osf.io/xftvq/
  dev_url: https://github.com/markhwhiteii/bwsTools
```